src/implementations/twig/components/link/package.json

```diff
@@ -9,22 +9,7 @@
     "@ecl/twig-component-icon": "^3.0.0"
   },
   "devDependencies": {
-    "@ecl/vanilla-component-link": "^3.0.0",
-    "@ecl/resources-ec-icons": "^3.0.0",
```

Comment:
`import uiIcons from '@ecl/resources-ec-icons/dist/lists/ui.json';` in the story file, so we need this dep.